README.md

```diff
@@ -1,84 +1,83 @@
+- If you have questions interpreting the results, please join us on Slack in #SIG-Windows.
 
+If you're new to building and testing Kubernetes, it's probably best to read the official [End-to-End Testing in Kubernetes](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md) page first.
 
-## Building Tests
+The rest of this page has a summary of those steps tailored for testing clusters that have Windows nodes.
 
-### e2e.test
+## Building the tests
 
-The official steps are in [kubernetes/community](https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#building-kubernetes-and-running-the-tests). For more details, be sure to read that doc. This is just a short summary.
+Testing windows is not different from regular testing of k8s clusters.  If you have never used a tool such as *e2e.test* or *sonobuoy* to run the Conformance (or other ginkgo based) test suites, then you should familiarize your self with that, first.
 
-Make sure you have a working [Kubernetes development environment](https://github.com/kubernetes/community/blob/master/contributors/devel/development.md) on a Mac or Linux machine. If you're using Windows, you can use WSL, but it will be slower than a Linux VM. The tests can be run from the same VM, as long as you have a working KUBECONFIG.
+Some resources introducing the e2e test framework:
 
-```bash
-go get -d k8s.io/kubernetes
-cd $GOPATH/src/k8s.io/kubernetes
-./build/run.sh make WHAT=test/e2e/e2e.test
-```
-Once complete, the binary will be available at: `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test`
+- https://kubernetes.io/blog/2019/03/22/kubernetes-end-to-end-testing-for-everyone/
+- https://github.com/kubernetes/community/blob/master/contributors/devel/sig-testing/e2e-tests.md#building-kubernetes-and-running-the-tests
 
-#### Cross-building for Mac or Windows
+Now, assuming you are able to build an e2e.test binary, we'll proceed with how windows the tests work.
 
-To build a binary to run on Mac or Windows, you can add `KUBE_BUILD_PLATFORMS`.
+**See the Questions section at the bottom of this doc** for pointers on how to build the e2e tests. 
 
-For Windows
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=windows/amd64 WHAT=test/e2e/e2e.test
-```
+## Running the e2e.test binary on a windows enabled cluster
 
-For Mac
-```bash
-./build/run.sh make KUBE_BUILD_PLATFORMS=darwin/amd64 WHAT=test/e2e/e2e.test
-```
-
-Your binaries will be available at `~/go/src/k8s.io/kubernetes/_output/dockerized/bin/linux/amd64/e2e.test` where `linux/amd64/` is replaced by `KUBE_BUILD_PLATFORMS` if you are building on Mac or Windows.
-
-
-## Running an e2e test pass
+If you already have a cluster, you will likely just need to build e2e.test, and run it with windows options.
 
+### Method 1: Using an existing cluster
```

Comment:
I think we should mention that you need to taint/label all linux nodes with node-role.kubernetes.io/master=NoSchedule.

This gets done by running this script on the master node during cluster proviosing
https://github.com/kubernetes-sigs/windows-testing/blob/c51f92045741c311d3e9a07eb00de79efddbb18f/extensions/master_extension/v1/win-e2e-master-extension.sh#L62

Reply:
ok done